### PR TITLE
[FT-229] My Accounts section on the iOS home screen

### PR DESCRIPTION
**Pull request by eldaroid**
Снимок экрана 2023-05-23 в 13 04 00

### CODE REVIEW

File: ios/OneCoin/Assets.xcassets/CardIcons/DebitCard.imageset/DebitCard.svg
Comment (VitalyPeryatin) [inline ios/OneCoin/Assets.xcassets/CardIcons/DebitCard.imageset/DebitCard.svg:1]: There is no need to add icons on the iOS side. All icons are already in the OneCoinShared library

File: ios/OneCoin/Core/UI/CoinTheme.swift [hunk 1]
```diff
@@ -7,18 +7,22 @@
...
     
-    init(colors: CoinColors, typography: Typography) {
+    public static var shared = CoinTheme.light
+    
+    private init(colors: CoinColors, typography: Typography) {
         self.colors = colors
         self.typography = typography
     }
+    
+    private func createCoinTheme() {
```
Comment (VitalyPeryatin) [inline ios/OneCoin/Core/UI/CoinTheme.swift:21]: We do we need a function with an empty body here?

File: ios/OneCoin/Core/UI/UIOneCoinConstants.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
+//
+//  UIOneCoinConstants.swift
+//  OneCoin
+//
+//  Created by Эльдар Попов on 22.05.2023.
+//
+
+import Foundation
+
+enum UI { }
```
Comment (VitalyPeryatin) [inline ios/OneCoin/Core/UI/UIOneCoinConstants.swift:10]: Why do we need an empty enum?

File: ios/OneCoin/Features/TabScreens/Accounts/Model/AccountCardModel.swift [hunk 3]
```diff
@@ -15,7 +15,19 @@ struct AccountCardModel: Identifiable {
...
-    .init(name: "Дебетовая карта", expences: 1000, color: .green),
-    .init(name: "Кредитная карта", expences: 10000, color: .red)
-]
+extension AccountCardModel {
+    static func getModelsMock() -> [AccountCardModel] { 
+        [
+            AccountCardModel(
+                name: "Debit card (*5841)", 
+                expences: 35000, 
+                color: Color(red: 75 / 255, green: 63 / 255, blue: 114 / 255)
```
Comment (VitalyPeryatin) [inline ios/OneCoin/Features/TabScreens/Accounts/Model/AccountCardModel.swift:24]: Can we get colors from OneCoinShared here?

File: ios/OneCoin/Features/TabScreens/Accounts/View/AccountsScreen.swift [hunk 4]
```diff
@@ -9,8 +9,12 @@ import SwiftUI
 
 struct CardAccountsScreen: View {
     var body: some View {
-        ForEach(accountCardModelsMock, id: \.id) { card in
-            AccountCard(with: card)
+        ScrollView(.horizontal, showsIndicators: false) {
```
Comment (VitalyPeryatin) [inline ios/OneCoin/Features/TabScreens/Accounts/View/AccountsScreen.swift:12]: There can be a lot of cards. Why don't we use `List` insteadOf `ScrollView` + `HStack`? I think that perfomance of `List` is better.

Comment (eldaroid) [reply ios/OneCoin/Features/TabScreens/Accounts/View/AccountsScreen.swift:12]: List doesn't support rendering and scrolling in a horizontal axis

File: ios/OneCoin/Features/TabScreens/Accounts/Model/AccountCardModel.swift (hunk 3 above)
Comment (eldaroid) [reply ios/OneCoin/Features/TabScreens/Accounts/Model/AccountCardModel.swift:24]: List doesn't support rendering and scrolling in a horizontal axis

File: ios/OneCoin/Core/UI/UIOneCoinConstants.swift (hunk 2 above)
Comment (eldaroid) [reply ios/OneCoin/Core/UI/UIOneCoinConstants.swift:10]: for further extensions like EU.Padding

File: ios/OneCoin/Core/UI/CoinTheme.swift (hunk 1 above)
Comment (eldaroid) [reply ios/OneCoin/Core/UI/CoinTheme.swift:21]: next PR

Comment (eldaroid) [reply ios/OneCoin/Core/UI/CoinTheme.swift]: corrected

**Review by VitalyPeryatin — APPROVED** [review]

**Pull request merged**